fairworkflows/fairworkflows/hello.py

```diff
@@ -0,0 +1,17 @@
+import ipywidgets as widgets
```

Comment:
For now I will leave it in because the search() widget still exists on the backend (slightly different from the frontend UI issue). It is now deprecated though - I will get rid of it soon.

Reply:
Oh sorry, you meant hello.py, no it is not needed anymore. Should be removed.

Reply:
It's been removed.